tsconfig.json

```diff
@@ -12,6 +12,7 @@
         "noImplicitReturns": true,
         "noImplicitThis": true,
         "noImplicitAny": true,
+        "skipLibCheck": true,
```

Comment:
Ohne das kompiliert flexsearch nicht - ist es bad practice das zu enablen?

Reply:
Nach neuesten next js configs (so wie wir es jetzt im freya haben) passt das so. Dort is das tsconfig autogenerated und da isses auch auf true. und der schreit beim build auch bei jedem Kleinkram. Also ich sage das passt so!